grr/server/grr_response_server/fleetspeak_utils.py

```diff
@@ -112,3 +113,22 @@ def GetLabelsFromFleetspeak(client_id):
       grr_labels.append(fs_label.label)
 
   return grr_labels
+
+
+def FetchClientResourceUsageRecordsFromFleetspeak(
```

Comment:
The module already contains "fleetspeak", I suggest to simplify this very long function name to: `FetchClientResourceUsageRecords`.

Reply:
I agree that it is quite trivial that it reflects retrieving data from Fleetspeak (and that this function name is long); on the other hand, it follows a similar convention that existed in the rest of the module, for instance:
https://github.com/google/grr/blob/b2969c7ed6f5c6e62919fa66625d992e7b5e0b34/grr/server/grr_response_server/fleetspeak_utils.py#L86-L87
What do you think?